### ISSUE TITLE
Displaying shapes using coordinates from Posenet in web editor also produces clicking sounds 

### ISSUE THREAD

**Issue by vidasdemo** [processing/p5.js-web-editor#928]
#### Nature of issue?

- Found a bug
- Existing feature enhancement
- New feature request

#### Details about the bug: 

- Web browser and version: 
- Operating System: 
- Steps to reproduce this bug:

#### New feature details:

**Comment by siddhant1** [comment]
Can you elaborate @vidasdemo ?

**Comment by vidasdemo** [comment]
Hi, 
This is my first post so apologies if my request is hard to follow!
I have just tried out the Posenet demo in ml5 for drawing a skeleton. The demo works as expected but in the p5 web editor each time a symbol is drawn which uses the coordinates provided by posenet a horrible sound is produced.  If I comment out the "draw skeleton" function and just request a symbol using hard coordinates there is no sound produced (as one would expect). This does not happen if the demo is viewed in a separate browser using the presentation link. I used the exact code in the demo

https://ml5js.org/docs/posenet-webcam

I referenced the latest Posenet in the Index.HTML file.

Can anyone provide an explanation? 
I asume this is something to do with the P5 web editor rather than Posenet.

thasnks

**Comment by vidasdemo** [comment]
Did you see the explanation? Was it clear enough?

 

cheers

**Comment by siddhant1** [comment]
Hey @vidasdemo , i thought you it was resolved from your side since you closed the issue , if it still persists kindly open it , I will have a look :)

**Comment by siddhant1** [comment]
Also can you confirm that the accessibility is turned off in the settings panel?

**Comment by vidasdemo** [comment]
Not resolved and its driving me nuts!

It’s my first post so not quite sure on mechanics. I did not mean to close the thread

**Comment by siddhant1** [comment]
Have you check the accessibility is turned of , you can press ctrl/CMD + shift +2 to turn off that too ,

**Comment by vidasdemo** [comment]
Problem solved!

 

Where can I find a reference to an explanation on what “accessibility”  does?

 

cheers

**Comment by siddhant1** [comment]
Awesome it got solved ! 
The accessibility settings are buried under settings panel .
Also many people are accidentally turning on accessibility settings , I have proposed a possible fix , waiting for @catarak views :)

**Comment by catarak** [comment]
@vidasdemo thanks for reporting, and sorry for the frustration! did you turn on the accessibility features, or were they just on? i think a lot of folks are turning them on accidentally.

**Comment by vidasdemo** [comment]
Hi Cassie,

I did not turn on the accessibility features…the “on” state seems to be the default when opening a new script.

Now I know about this “feature” I can switch it off.

My suggestion would be to add an explanation of what “accessibility” is about somewhere in the getting started /reference section. 

It’s one thing to know the “cure” but it would also be nice to know something about the “condition”!!

 

Cheers

 

Vidas

**Comment by catarak** [comment]
@vidasdemo that's helpful. it's not on by default, but what happens is that it gets turned on, and then that setting is saved to your account.
